deploy/helm/sumologic/conf/events/otelcol/config.yaml

```diff
@@ -39,6 +39,8 @@ processors:
     source_name: {{ .Values.fluentd.events.sourceName | quote}}
     source_category: {{ .Values.fluentd.events.sourceCategory | default (printf "%s/%s"  (include "sumologic.clusterNameReplaceSpaceWithDash" .) (.Values.fluentd.events.sourceName )) | quote}}
     source_category_prefix: ""
+  sumologic_schema:
+    add_cloud_namespace: false
```

Comment:
Don't you need to set `translate_attributes: false` on the `sumologic` exporter?

Reply:
It should be idempotent and we can skip additional changes when they will be dropped.